web/sites/default/settings.php

```diff
@@ -71,6 +71,12 @@
       $settings['container_yamls'][] = $app_root . '/' . $site_path . '/services.yml';
       break;
 
+    case 'D8NID-edge':
+      // Retain as much prodution related config/settings as possible.
```

Comment:
spelling mistake here 'prodution' - very picky and tiny I know !

Reply:
Good spot, thanks!